app/models/podcast.rb

```diff
@@ -52,6 +56,33 @@ def explicit=(value)
     super(EXPLICIT_ALIASES[value] || value)
   end
 
+  def itunes_category
+    itunes_categories.first&.name
+  end
+
+  def itunes_category=(value)
```

Comment:
Opted to create some getters/setters for these, and only support updating 1 category / 1 subcategory in the form.